src/backend/base/langflow/api/utils.py

```diff
@@ -283,6 +283,7 @@ async def cascade_delete_flow(session: AsyncSession, flow_id: uuid.UUID) -> None
     try:
         await session.exec(delete(TransactionTable).where(TransactionTable.flow_id == flow_id))
         await session.exec(delete(VertexBuildTable).where(VertexBuildTable.flow_id == flow_id))
+        await session.exec(delete(MessageTable).where(MessageTable.flow_id == flow_id))
```

Comment:
I'm not sure this is a good idea. Even though we tie the message to a flow and if we set it to null it won't appear in any playground anymore, the session id, which is what we use to search for messages might still be used. 

Could you comment this and add a TODO here for us to verify this behavior later?

Reply:
Thank you for the feedback. I have added a comment and a TODO. Please review the changes.